.github/styles/Bokeh/WordList.yml

```diff
@@ -0,0 +1,41 @@
+extends: substitution
+message: "Use '%s' instead of '%s' (if possible)."
+level: error
+ignorecase: false
+action:
+  name: replace
+swap:
+  '(?:API Console|dev|developer) key': API key
+  '(?:e-mail|Email|E-mail)': email
```

Comment:
I think "e-mail" should be fine (at least Wikipedia agrees). I never used "email" myself.

Reply:
I'm basically OK with either, but we should be consistent. "email" is the spelling recommended by the Goolge developer documentation style guide (https://developers.google.com/style/word-list#letter-e) that I referenced in the documentation style guidelines for Bokeh. So it might be the easiest solution to just stay consistent with their recommendation. Also, I don't think that "email" will be used a lot in Bokeh's docs anyways (neither "e-mail" nor "email" are currently mentioned in the docs). So, I'd like to keep the original version here. 
Any thoughts here, @bryevdv?

Reply:
I don't have a preference re: docs except consistency, i will note in passing that I have never personally used "e-mail" and only ever use "email" myself.